README.adoc

```diff
@@ -2876,6 +2878,20 @@ the var as documentation.
   (bar x))
 ----
 
+=== Docstrings and Protocols [[docstrings-and-protocols]]
+
+Unlike `defn`, docstrings for `defprotocol` methods are placed _after_ the argument vector:
```

Comment:
Probably the guideline should be to document `defprotocol` methods and this should be just a `NOTE:` after the guideline. Otherwise it just sounds a bit weird in the context of a style guide. :-)

Reply:
Or just start the sentence with "Place". :-)

Reply:
Thanks for reviewing! Yes, I agree.

I think you mean we could adjust defprotocol section to instead be a `NOTE:` on `defn` guidance?  We just told the reader something but, look,  here's an exception for defprotocol.

I'll try that out.

Reply:
Ok @bbatsov, pushed change, how does that read/feel to you?

Happy to try more alternatives, lemme know.